autofit/aggregator/summary/aggregate_fits.py

```diff
@@ -27,7 +28,7 @@ class FITSFit(Enum):
     The HDUs that can be extracted from the fit.fits file.
     """
 
-    ModelImage = "MODEL_IMAGE"
+    ModelData = "MODEL_IMAGE"
```

Comment:
It's less confusing if name and value contain the same words but not absolutely necessary